comicScraper.py

```diff
@@ -147,30 +159,48 @@ def main():
         # TODO: add a check here for /special/areyouhuman
         # TODO: kick into a chrome window and wait for user input
         imageLinks.append(issueImageLinks)
-        issueImageDict[issueName] = issueImageLinks
+
+        if singleIssue:
+            # comicTitle contains the issue name in this case
+            issueImageDict[comicTitle] = issueImageLinks
+        else:
+            issueImageDict[issueName] = issueImageLinks
+
+        # This counter could probably be tweaked for faster performance    
         counter=random.randint(10,20)
         print(f"Sleeping for {counter} seconds")
         time.sleep(counter)
     print(f"Image links: {' '.join(map(str, imageLinks))}")
 
-    # uses the list object to package all the images into a single CBZ
-    for issue in imageLinks:
-        saveImagesFromImageLinks(issue)
-    folderCBZPacker(comicTitle)
+    # Determine length of full comic (how many zeroes to pad)
+    if downloadFull:
+        for key in issueImageDict:
+            comicLength += len(issueImageDict[key])
+
+        # uses the list object to package all the images into a single CBZ
+        for issue in imageLinks:
+            saveImagesFromImageLinks(issue, comicLength)
+        folderCBZPacker(comicTitle)
 
-    # uses the dict object to package the images into multipul CBZs
-    for key in issueImageDict:
-        global COUNTER
-        COUNTER = 1
-        path = saveImagesFromImageLinks(issueImageDict[key], key)
-        folderCBZPacker(comicTitle, key)
+    else:
+        # uses the dict object to package the images into multiple CBZs
+        for key in issueImageDict:
+            global COUNTER
+            COUNTER = 1
+            path = saveImagesFromImageLinks(issueImageDict[key], len(issueImageDict[key]), key)
+            if singleIssue:
+                folderCBZPacker(comicTitle, "")
+            else:
+                folderCBZPacker(comicTitle, key)
+    
 
 if __name__ == "__main__":
     # build the parser
     parser = argparse.ArgumentParser(description='Script for downloading CBZ files from readcomiconline.li',
     epilog='Example: comicScraper.py https://readcomiconline.li/Comic/Sandman-Presents-Lucifer')
     parser.add_argument('URL', help='The url of the comic to download')
     parser.add_argument('-f', '--folder', help='The folder to save the comic in')
+    parser.add_argument('-c', '--complete', help='Download the entire comic into one folder. Omit this argument to download each issue into its own folder')
```

Comment:
we should change this so just the flag is needed `-c / --complete` no need to add the `Y`
```
parser.add_argument('-c', '--complete', help='Download the entire comic into one folder. Omit this argument to download each issue into its own folder', action='store_true')
```

Reply:
Updated